Update Example .travis.yml in CI Guide

mtmckenna opened the pull request:
Installing applesimutils seems to be required to have builds pass in Travis.

LeoNatan commented on docs/Guide.RunningOnCI.md:
```diff
@@ -68,6 +68,8 @@ env:
 
 install:
+- brew tap wix/brew
+- brew install --HEAD applesimutils
```
Please do no longer use `—HEAD`. There are proper versions now, which should be stable.

mtmckenna replied:
Thank you! Updated the pull request.

mtmckenna commented:
Based on feedback from @LeoNatan , I removed the `--HEAD` argument from the `brew` command. I also removed `--HEAD` from the same command I found elsewhere in the docs.

I left the `--HEAD` in `scripts/install.ios.sh` because I didn't want this pull request to modify any code. However, perhaps it makes sense to make the change to `.install.ios.sh` as well in a different commit?

mtmckenna commented:
Hello! Anything I should do to help get this one merged in? Thank you!
